site/layouts/partials/series_link.html

```diff
@@ -0,0 +1,40 @@
+{{ if .Params.series }}
+    {{ $.Scratch.Add "cur_page_num" 1 }} 
+    {{ $.Scratch.Add "total_page_num" 0 }} 
+    {{ range where .Site.RegularPages.ByDate "Params.series" .Params.series }} 
+        {{ $.Scratch.Add "total_page_num" 1 }} 
+        {{ if gt $.Date.Unix .Date.Unix }} 
+            {{ $.Scratch.Add "cur_page_num" 1 }} 
+            {{ $.Scratch.Set "prev_link" .Permalink }}
+            {{ $.Scratch.Set "prev_title" .Title }}  
+        {{ end }}
+    {{ end }}
+    {{ range where .Site.RegularPages.ByDate.Reverse "Params.series" .Params.series }} 
+        {{ $.Scratch.Set "first_link" .Permalink }}
+        {{ if lt $.Date.Unix .Date.Unix }}
+            {{ $.Scratch.Set "next_link" .Permalink }}
+            {{ $.Scratch.Set "next_title" .Title }}
+        {{ end }}
+    {{ end }}
+    {{ if or ($.Scratch.Get "next_link") ($.Scratch.Get "prev_link") }} 
+        <hr/>
```

Comment:
This line looks weird, as it doesn't align the line of the left. What was the reason for adding it?
![Screenshot from 2020-03-13 11-16-53](https://user-images.githubusercontent.com/16541325/76612352-6e2aa680-651c-11ea-92c4-f815ac06371c.png)